### PR TITLE
Add renamed CopilotAgent commands to README.md 

### PR DESCRIPTION
**Pull request by ziontee113**
## WHAT

Add renamed CopilotAgent commands to README.md 

## WHY

Discussed in #27

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +35,10 @@ return {
     keys = {
       { "<leader>cce", "<cmd>CopilotChatExplain<cr>", desc = "CopilotChat - Explain code" },
       { "<leader>cct", "<cmd>CopilotChatTests<cr>", desc = "CopilotChat - Generate tests" },
+
+      { "your_keymap", ":CopilotChatVsplit", mode = "n", desc = "CopilotChat - Open in vertical split in normal mode" },
```
Comment (jellydn) [inline README.md:39]:
```suggestion
      { "<leader>ccv", ":CopilotChatVsplit", mode = "n", desc = "CopilotChat - Open in vertical split in normal mode" },
```

File: README.md [hunk 2]
```diff
@@ -31,6 +35,10 @@ return {
...
+      { "your_keymap", ":CopilotChatVsplitVisual", mode = "x", desc = "CopilotChat - Open in vertical split in visual mode" },
```
Comment (jellydn) [inline README.md:40]:
```suggestion
      { "<leader>ccv", ":CopilotChatVsplitVisual", mode = "x", desc = "CopilotChat - Open in vertical split in visual mode" },
```

File: README.md [hunk 3]
```diff
@@ -31,6 +35,10 @@ return {
...
+      { "your_keymap", "<cmd>CopilotChatInPlace", mode = "x", desc = "CopilotChat - Open in-place code replacement layout" },
```
Comment (jellydn) [inline README.md:41]:
```suggestion
      { "<leader>ccx", "<cmd>CopilotChatInPlace", mode = "x", desc = "CopilotChat - Open in-place code replacement layout" },
```

**Review by jellydn — APPROVED** [review]
Thanks, @ziontee113 LGTM!

**Comment by jellydn** [comment]
@ziontee113 Could you add demo GIF for the new commands on readme? It would be great if you could follow my convention (e.g: add step, record video).  🙏

**Comment by ziontee113** [comment]
That's good. I'll send a PR tomorrow.

**Pull request merged**